pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["maturin>=0.12,<0.13"]
+build-backend = "maturin"
```

```suggestion
build-backend = "maturin"

[tool.maturin]
sdist-include = ["Cargo.lock"]
```

c.f. https://github.com/PyO3/maturin/discussions/748#discussioncomment-1840191